package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "beachball": "^1.32.0",
+    "clang-format": "1.5.0",
```

Comment:
Dependencies in the root package.json are reserved for bits that don't belong to any particular package. We want to keep this as a dependency of just the format-files script, since that what consumes it.

E.x. imagine if we decided to globally include a library into every liblet instead of declaring only dependencies.

Reply:
I will remove it then. I was not sure what the rules are.